guides/MAC_SETUP.md

```diff
@@ -130,6 +130,16 @@ Then follow [instructions to add the SSH key in the clipboard to your github acc
 
 Return to the [Getting Started](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md) guide for next steps.
 
+### Testing SSH Connection
+
+To check that you've setup your Git SSH on your terminal open your favoured terminal and enter:
+  * `ssh -T git@github.com`
+  
+You will then be asked to type *yes* if you're ready to connect.  You should then be then be prompted with:
```

Comment:
- Double `then be` we can remove one
- Removal of double spaces (think the rest of our document uses single space after full-stop so just a consistency thing although optional)
```suggestion
You will then be asked to type *yes* if you're ready to connect. You should then be prompted with:
```